.github/workflows/nextjs_bundle_analysis.yml

```diff
@@ -54,7 +55,7 @@ jobs:
          # Here's the first place where next-bundle-analysis' own script is used
          # This step pulls the raw bundle stats for the current bundle
          - name: Analyze bundle
-           run: npx -p nextjs-bundle-analysis@0.5.0 report
+           run: pnpm --package=nextjs-bundle-analysis@0.5.0 dlx report
```

Comment:
What's the reasoning for this change?

Reply:
Mainly for consistency. We are running `pnpm install` in an earlier step but we are then using `npm exec` here.